Исключения для артефактов в Лиманске

ISTORMI opened the pull request:
На локе Лиманск осталось множество домов куда невозможно попасть игроку, при этом там бывают артефакты.
Новые места для исключений:
гастроном и подвал
дом, напротив входа в подвал гастронома
дом, слева от входа в дом сталкеров
дом, напротив беседки где 2 мус. бака
2й этаж дома, слева от почтовых ящиков
2й этаж дома, слева от мусорки с тайником

731 ; Крыша дома за локацией -- Поправил, чтоб цепляло аи-сетку за локой.

ISTORMI committed:
Перенос текстовых строк из akill.script в xml

Цвет для СМС берем из color_defs.xml
Замена всех слов "ПДА" на "КПК"

dsh2dsh committed:
Merge branch 'master' into master

ISTORMI committed:
Убрал лишнюю строку

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Перенос текстовых строк из папки op2ogse\gamedata\scripts\battle\ в xml

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Перенос всех строк связанных с ui в единый ui.xml

Все строки связанные с UI из всех файлов в папке text\rus переехали в единый ui.xml тут же будут и все остальные ui строки текста. Мусор от мультиплеера остался, его можно удалить\перенести отдельно для удобства, либо оставить как есть.
Мелкие корректировки разного неактуального текста и не только, почищены повторы. Пересмотр и добавление описаний для некоторых доп. настроек видео, такие как сглаживание, тесселяция и т.д.
По причине выше полностью пересмотрен ui_mm_opt_video_adv.script но настройки которые кривые, либо не работают - остались. См. внутри комменты в строках: 41, 45 и 52

stable_dialogs_amk.xml - форматирование\перенос текста

ISTORMI committed:
Revert "Перенос всех строк связанных с ui в единый ui.xml"

This reverts commit 4956a0039633f279322dc8bf40bc77c0e99a0c01.

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Перенос текстовых строк из buusty_dialog.script в xml

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Перенос текстовых строк из oksana_spawn.script в xml

string_table_oksana579.xml - Перенос смс строк\форматирование

ISTORMI committed:
Merge branch 'master' of https://github.com/ISTORMI/op2ogse

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Merge branch 'dsh2dsh:master' into master

ISTORMI committed:
Перенос текстовых строк из динамических новостей в xml

ISTORMI committed:
Merge branch 'master' of https://github.com/ISTORMI/op2ogse

ISTORMI committed:
Исключения для артефактов в Лиманске

На локе Лиманск осталось множество домов куда невозможно попасть игроку, при этом там бывают артефакты.
Новые места для исключений:
гастроном и подвал
дом, напротив входа в подвал гастронома
дом, слева от входа в дом сталкеров
дом, напротив беседки где 2 мус. бака
2й этаж дома, слева от почтовых ящиков
2й этаж дома, слева от мусорки с тайником

731 ; Крыша дома за локацией -- Поправил, чтоб цепляло аи-сетку за локой.

dsh2dsh commented on gamedata/config/scripts/amk/anoms/anoms.ltx:
```diff
@@ -752,6 +752,18 @@
 
+ -5.45, -0.85, -148.01, 25 ; ��������� � ������
+-17.87,  1.41, -180.99, 10 ; ���, �������� ����� � ������ ����������
+-27.78,  1.41, -180.99, 10 ; ---
```
Тут можно было сделать одно исключение с большим радиусом, вместо двух?

ISTORMI replied:
Прописать координаты исключения в игру можно только в виде куба. Если сделать 1 - то этот куб станет огромный, прятать его некуда, будет значительная потеря аи-сетки, соот-но и потеря мест для артов и аномалий. Т.ч. по желанию, можно и 1.

dsh2dsh replied:
Почему в виде куба. Здесь указываются сферы. Если одной сферой можно накрыть здание, то так и нужно, вместо того, чтобы указывать две сферы для каждого из этажей, к примеру.

ISTORMI replied:
Убрал все кубы и заменил их на сферы, действительно, сферы достаточно одной.

dsh2dsh replied:
Как ты вообще эти кубы указывал? Ведь задаются x, y, z и радиус. Как это может быть кубом?

ISTORMI replied:
![Снимок222](https://user-images.githubusercontent.com/75955587/198877048-da0d3457-2f90-4b79-9f5f-4ac611fe3e1e.PNG)
Подумав что игра будет ставить куб - это только мой косяк и невнимательность

ISTORMI committed:
Update anoms.ltx

Правильно делать сферу а не куб, для координат исключений артефактов\аномалий
куб > сфера
